web/themes/joinup/js/form.js

```diff
@@ -45,6 +45,9 @@
       $(context).find('form').once('ajaxReload').each(function () {
         $(document).ajaxComplete(function (event, xhr, settings) {
           componentHandler.upgradeAllRegistered();
+          $('.mdl-js-checkbox').each(function (index, element) {
```

Comment:
It is accidentally added commit from ISAICP-3237, could we leave it as it is to avoid merge conflicts?